docs/header.sticky.md

```diff
@@ -1,12 +1,35 @@
 # qgrid 
+
+## Option 1 - settings in template
```

Comment:
Will we put all setting to template or configure it from controller? In my mind it would be good in controller to keep markup more simple and readable. And use markup to customise templates only.
So I suggest 2 options below.